pkg/recipes/terraform/config/providers/aws.go

```diff
@@ -127,7 +136,10 @@ func fetchAWSCredentials(ctx context.Context, awsCredentialsProvider credentials
 			return nil, nil
 		}
 	case ucp_datamodel.AWSIRSACredentialKind:
-		return nil, errors.New("AWS IRSA Credential is not supported yet.")
+		if credentials.IRSACredential == nil || credentials.IRSACredential.RoleARN == "" {
```

Comment:
```suggestion
		if credentials.IRSACredential == nil && credentials.IRSACredential.RoleARN == "" {
```
I wonder if this should be `and` operator here.